src/components/navigator/Navigator.tsx

```diff
@@ -107,11 +111,12 @@ const Navigator = () => {
     pathname !== SHARED_ITEMS_PATH &&
     pathname !== HOME_PATH
   ) {
-    return <Navigate to={HOME_PATH} />;
+    return null;
```

Comment:
You don't redirect anymore?

Reply:
No. I currently return an empty content.